src/pages/paras/[user_id]/index.tsx

```diff
@@ -12,19 +14,63 @@ const ViewParaPage = () => {
     { enabled: Boolean(user_id) }
   );
 
+  const { mutate } = trpc.unassignPara.useMutation();
+
+  const archivePara = async () => {
+    if (!para) {
+      return;
+    }
+    mutate({ para_id: para.user_id });
+    await router.push(`/cmDashboard`);
+  };
+
   if (isLoading) {
     return <div>Loading...</div>;
   }
 
   return (
     <div className={styles.person}>
-      <Link href={`/cmDashboard`}>
-        <p>CM Dashboard</p>
-      </Link>
-      <h1>Para {para?.user_id}</h1>
-      <p>
+      <h1>
         {para?.first_name} {para?.last_name}
+      </h1>
+      <p>
+        <b>Para ID:</b> {para?.user_id}
       </p>
+      <p>
+        <b>Para Email:</b> {para?.email}
+      </p>
+      <button
+        className={`${styles.signIn} ${styles.bold}`}
+        onClick={() => setArchive(true)}
+      >
+        {" "}
+        Archive Para
```

Comment:
s/Archive/Remove/g

what does "archiving" a para mean? "Remove para" is the natural opposite of "Add para"

Reply:
We're not deleting the para, only unassigning the para with the CM here. In our backend we name the function "unassignPara". I think the reason we put "archive para" in the frontend was maybe because saying Remove Para is misleading because we aren't deleting the para. Should we say "Unassign Para" instead?

Reply:
We are "deleting" the para from the CM's list . Archive has a connotation of storing something such the archived entity can be later retrieved/un-archived. That is not the case here.

In general, the backend model shouldn't leak into the FE user experience.

Reply:
I changed the state to `unassignPrompt` and fixed all the code to refer to `unassign` instead of `archive`.